charts/enterprise-metrics/capped-large.yaml

```diff
@@ -0,0 +1,144 @@
+# These values configure the Grafana Enterprise Metrics cluster to
+# handle production ingestion of ~10M active series using the blocks
+# storage engine.
+# Query requirements can vary dramatically depending on query rate and query
+# ranges. The values here satisfy a "usual" query load as seen from our
+# production clusters at this scale.
+# It is important to ensure that you run no more than one ingester replica
+# per node so that a single node failure does not cause data loss. Zone aware
+# replication can be configured to ensure data replication spans availability
+# zones. Refer to [Zone Aware Replication](https://cortexmetrics.io/docs/guides/zone-aware-replication/)
+# for more information.
+# Minio is no longer enabled and you are encouraged to use your cloud providers
+# object storage service for production deployments.
+admin_api:
+  replicas: 3
+  resources:
+    requests:
+      cpu: 1
+      memory: 256Mi
+    limits:
+      cpu: 1
+      memory: 256Mi
+
+alertmanager:
+  persistentVolume:
+    enabled: true
+  replicas: 3
+  resources:
+    requests:
+      cpu: 1
+      memory: 8Gi
+    limits:
+      cpu: 1
+      memory: 8Gi
+  statefulset:
+    enabled: true
+
+compactor:
+  persistentVolume:
+    size: 50Gi
+  resources:
+    requests:
+      cpu: 2
```

Comment:
Similar question as above for the admin_api requests. `1` -> `2`. Additionally, I'm not sure if it is worth setting this to `2` CPUs if we don't increase compactor concurrency too.

It's also drawn my attention to the compactor PV size which should probably be close to 150Gi given the goal to handle 10M active series (https://cortexmetrics.io/docs/blocks-storage/compactor/). This is dependent on the size of the largest tenant but 150Gi should cover us even if it is only a single tenant.

Reply:
limit was increased from `1200m` to `2` for the same reason as stated above.
therefore also the requests changed from `1` to `2`